bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/AbstractZkLedgerManager.java

```diff
@@ -179,8 +179,16 @@ protected AbstractZkLedgerManager(AbstractConfiguration conf, ZooKeeper zk) {
 
     @Override
     public void process(WatchedEvent event) {
-        LOG.info("Received watched event {} from zookeeper based ledger manager.", event);
+        LOG.debug("Received watched event {} from zookeeper based ledger manager.", event);
         if (Event.EventType.None == event.getType()) {
+            if (Event.KeeperState.Expired == event.getState()) {
+                LOG.info("ZooKeeper client expired on ledger manager.");
```

Comment:
Maybe it would be useful to log the session id, with the prev code we had the event, now       the event is logged at debug level. Don't know if event.toString contained the session id..
Anyway while analyzing zk session expiration events it is useful to have the id

Reply:
Event.toString doesn't contain any session information.

It is also a bit tricky to log session id as well. because the underneath retry logic in the zookeeper client we are using. at the time the watcher callback was trigger, the underneath client might already reconnect and a new session id is available. if we try to log the session id, it might be actually misleading. 

If we want to achieve accurate logging with session id, we have to refactor the way how we submit callback, which I think it is an excessive change here.

Reply:
Clear. Okay